cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterServiceMain.java

```diff
@@ -481,96 +487,28 @@ private void updateConfigurationTable() {
     }
   }
 
-  /**
-   * The replication Status Tool coprocessors will use replication state table to track
-   * replication status. Create the table here.
-   */
-  private void createReplicationStateTable() throws IOException {
-    LOG.info("Create replicationstate table now.");
-    HBaseTableUtil tableUtil = new HBaseTableUtilFactory(cConf).get();
-    HBaseAdmin admin = new HBaseAdmin(hConf);
-    try {
-      TableId tableId = getReplicationStateTableId(tableUtil);
-      HTableDescriptorBuilder htd = tableUtil.buildHTableDescriptor(tableId);
-      htd.addFamily(new HColumnDescriptor(Constants.ReplicationStatusTool.WRITE_TIME));
-      htd.addFamily(new HColumnDescriptor(Constants.ReplicationStatusTool.REPLICATE_TIME));
-      htd.addFamily(new HColumnDescriptor(Constants.ReplicationStatusTool.SHUTDOWN_TIME));
-      tableUtil.createTableIfNotExists(admin, tableId, htd.build());
-      // Clear shutdown row to keep shutdown time for current instance only
-      saveShutdownTime(0L);
-    } catch (IOException e) {
-      throw Throwables.propagate(e);
-    } finally {
-      admin.close();
-    }
-    LOG.info("replicationstate table created");
-  }
-
-  private TableId getReplicationStateTableId(HBaseTableUtil tableUtil) throws IOException {
-    String tableName = hConf.get(Constants.ReplicationStatusTool.REPLICATION_STATE_TABLE_NAME);
-    String ns = hConf.get(Constants.ReplicationStatusTool.REPLICATION_STATE_TABLE_NAMESPACE);
-
-    TableId tableId = tableUtil.createHTableId(
-      (ns != null)
-        ? new NamespaceId(ns)
-        : new NamespaceId(Constants.ReplicationStatusTool.REPLICATION_STATE_TABLE_DEFAULT_NAMESPACE),
-      (tableName != null) ? tableName : Constants.ReplicationStatusTool.REPLICATION_STATE_TABLE_DEFAULT_NAME);
-
-    return tableId;
+  private static boolean isReplication(final Configuration c) {
```

Comment:
hbase.replication is an existing setting that is required for hbase replication to work, not like introduced for the tool only. In any case, I will remove the check.